ovgs.proto

```diff
@@ -1,175 +1,254 @@
 // Copyright (c) 2023 Arista Networks, Inc.
 // Use of this source code is governed by the BSD 2-Clause License
 // that can be found in the LICENSE file.
-
 syntax = "proto3";
 
-import "google/protobuf/timestamp.proto";
-
 package ovgs.v1;
 
-option go_package = "arista/sztp/ovgs/gen";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/aristanetworks/ownership-voucher-grpc/ovgs";
 
-enum userRole {
-    UNSPEC = 0;
-    SUPPORT = 1;
-    ADMIN = 2;
-    REQUESTOR = 3;
+enum UserRole {
+  USER_ROLE_UNSPECIFIED = 0;
+  USER_ROLE_SUPPORT = 1;      // Read write assocations and certs
+  USER_ROLE_ADMIN = 2;        // Read write to everything
```

Comment:
The SUPPORT role was intended as an internal (to the vendor who implements this interface) to allow taking SUPPORT like operations. A comment below mentions it, but USER_ROLE_ADMIN and USER_ROLE_REQUESTOR is what client of the service would generally use